hackathon_site/hackathon_site/tests.py

```diff
@@ -25,6 +26,20 @@ def setUp(self):
     def _login(self):
         self.client.login(username=self.user.username, password=self.password)
 
+    @staticmethod
+    def _apply_to_profile_as_user(user, team=None):
```

Comment:
Why `_apply_to_profile`? What does this have to do with applying? Perhaps `_make_profile` would be a better name

Reply:
changed